lib/datadog/core/diagnostics/health.rb

```diff
@@ -30,6 +37,26 @@ class Metrics < Core::Metrics::Client
           gauge :queue_max_length, Ext::Health::Metrics::METRIC_QUEUE_MAX_LENGTH
           gauge :queue_spans, Ext::Health::Metrics::METRIC_QUEUE_SPANS
           gauge :sampling_service_cache_length, Ext::Health::Metrics::METRIC_SAMPLING_SERVICE_CACHE_LENGTH
+
+          def default_metric_options
```

Comment:
Is this public method tested somewhere?

I supposed it would be a better candidate for verifying `service` tag, compared a more integrated `#api_requests` test.